manifests/i/IObit/AdvancedSystemCare/16.4.0.225/IObit.AdvancedSystemCare.installer.yaml

```
@@ -0,0 +1,25 @@
+# Automatically updated by the winget bot at 2023/May/11
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: IObit.AdvancedSystemCare
+PackageVersion: 16.4.0.225

```

```suggestion
PackageVersion: 16.4.0
```
Incorrect package version